geolocation/android/src/main/java/com/capacitorjs/plugins/geolocation/GeolocationPlugin.java

```diff
@@ -171,11 +173,15 @@ public void error(String message) {
 
     @SuppressWarnings("MissingPermission")
     private void startWatch(final PluginCall call) {
-        int timeout = call.getInt("timeout", 10000);
+        long timeout = call.getLong("timeout", 10000L);
+        long interval = call.getLong("interval", 5000L);
+        long minInterval = call.getLong("minInterval", IMPLICIT_MIN_UPDATE_INTERVAL);
```

Comment:
According to android docs, `IMPLICIT_MIN_UPDATE_INTERVAL` is the default value for `setMinUpdateIntervalMillis` (`-1`)